cypress/integration/todo_item_spec.js

```diff
@@ -0,0 +1,39 @@
+/// <reference types="cypress" />
```

Comment:
What's this?

Reply:
this turns on IntelliSense for Cypress type in JS specs in VSCode for example

![intellisense-vscode](https://user-images.githubusercontent.com/2212006/37596173-b47d6c8c-2b51-11e8-9912-f3e7403811b1.jpeg)